DxDispatch/src/dxdispatch/CommandLineArgs.cpp

```diff
@@ -112,10 +116,19 @@ CommandLineArgs::CommandLineArgs(int argc, char** argv)
         m_showAdapters = result["show_adapters"].as<bool>(); 
     }
 
-    if (result.count("direct_queue") && result["direct_queue"].as<bool>()) 
-    { 
+    auto queueTypeStr = result["queue_type"].as<std::string>();
```

Comment:
[naming nit] If we're spelling out string in `m_adapterSubstring`, then spelling out string in `queueTypeString` makes sense.